### ISSUE TITLE
Error occurred during setter call: No setter called for key 

### ISSUE THREAD

**Issue by james-woods** [nielsbasjes/logparser#47]
In Snapshot 4.1 - after using the new `setterPolicy = NOT_NULL` on a field like:

```java
@Field(value = "NUMBER:connection.client.logname", setterPolicy = NOT_NULL)
  public void setRemoteLogName(String remoteLogName) {
    this.remoteLogName = remoteLogName;
  } 
```
the following error is thrown, when the field is `-` (`null`) in the log that gets processed:
```
Error occurred during setter call: No setter called for  key = "NUMBER:connection.client.logname"  name = "NUMBER:connection.client.logname"  value = "Value{filled=STRING, s='null', l=null, d=null}"
```

**Comment by nielsbasjes** [comment]
Great catch! 
My tests were in adequate.
Please check if it is fixed now in the 4.1-SNAPSHOT release.

**Comment by james-woods** [comment]
Fixed :)
